lib/shopify_transporter/exporters/magento/customer_exporter.rb

```diff
@@ -12,14 +12,21 @@ def initialize(store_id: nil, soap_client: nil, database_adapter: nil)
 
         def export
           $stderr.puts 'Starting export...'
-          base_customers.map do |customer|
-            $stderr.puts "Fetching customer: #{customer[:customer_id]}..."
-            customer.merge(address_list: customer_address_list(customer[:customer_id]))
+          puts '['
```

Comment:
Man, this entire thing is such a hack. I don't know of an alternative way to do this, though.

Reply:
Preaching to the choir. I'd love any alternative way to do this. At all.

It hurt me to write this code.

Reply:
I think a next step would be subsequent refactors where we strip out the duplicate code between each of these Exporter#export methods. We would then have a service object that does this printing to stdout and we can just feed it blocks to print or something. I'll play around with it, I think it would be a nice way to just ensure we've taken all the...pragmatic stuff and encapsulated somewhere.

Reply:
Sounds like a good plan.